client/checkout/utils/upe.js

```diff
@@ -300,3 +300,71 @@ export const getStripeElementOptions = (
 
 	return options;
 };
+
+/**
+ * Converts form fields object into Stripe `shipping` object.
+ *
+ * @param {Object} fields Object mapping checkout shipping fields to values.
+ * @return {Object} Stripe formatted `shipping` object.
+ */
+export const getShippingDetails = ( fields ) => {
```

Comment:
@nagpai: I can see the phone is removed compared to the original functions. Is it intended?

Reply:
Intentionally not included, since the [sample information](https://stripe.com/docs/payments/afterpay-clearpay/accept-a-payment?platform=web&ui=checkout#accept-a-payment) does not include phone, indicating its not needed by Afterpay . 

The classic checkout fields, as well as form submit data, does not include a phone number for shipping. 

<img width="520" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/4162931/efa057fd-f9ed-4e56-b7dc-5ead5f579a5d">

<img width="486" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/4162931/db9d7d6b-4d34-4092-874b-f5a858c8d51d">